documentation/release-notes/topics/version_4.rst

```diff
@@ -2,12 +2,90 @@
+
+- Administrators can now select a notification channel for an Alarm Policy; more information on Notification Channels is below.
+
+========================================================
+Enhanced Feature: Notifications -> Notification Channels
+========================================================
+- This feature allows Administrators to create an Alarm Policy to channel notifications to an external communication or alerting system such as Microsoft Teams.
+
+- This release adds the integration with Microsoft Teams. Administrators can configure a link to a specific Microsoft Teams channel. Administrators can add multiple channels and target them with different Alarm Policies.
+
+================================================
+Enhanced Feature: Linux & Windows Data Collector
+================================================
+- VMware protocol has been enhanced to add a monitoring pipeline for discovered sensors. To use this enhancement, customers must update to the latest version of the Data Collector and rediscover the assets.
+
+======================================
+Other notable changes and improvements
+======================================
+- Data grids will save column selection and sorting order by default.
+
+- Launch Web Interface has been added as a primary action button on device asset types.
+
+- Volume Unit has been added as a :ref:`locale setting <Locale-settings-doc>` in the application.
+
+- The Documents section has been moved to be a main navigation menu item. Previously, it was under the Assets section.
+
+- The location layout editor now supports adding triangular shapes.
+
+- Multi-value asset properties like serial numbers, IP addresses, and MAC addresses have been updated to have consistent sorting order in search results and other display contexts, where applicable.
+
+=================
+Notable bug fixes
+=================
+- **AS-11359** Fixed a bug in the Debian Linux SSH protocol definition that caused storage capacity sensors not to be updated during the monitoring cycle.
+
+- **AS-13941** Fixed a bug that, under certain conditions, caused the page not to render with search results when navigating from the asset summary widget to Advanced Search.
+
+- **AS-14086** Fixed a bug that caused user password resets to fail under certain conditions.
+
+- **AS-14107** Fixed a bug that, under certain conditions, caused the breaker information of certain Eaton Large PDUs to be discovered incorrectly.
+
+- **AS-14252** Fixed a bug that caused authentication to fail when discovering IxOS-based devices.
+
+- **AS-14306** Fixed a bug that, under certain conditions, caused a sensor threshold alarm event to be closed and opened. While the result was the same, it caused extra logging and, in some cases, extra notifications. The bug fix will close any open sensor threshold alarm event where applicable. The system will re-evaluate and open new alarm events where needed with the next monitoring cycle.
```

Comment:
"... The bug fix will close any open sensor threshold alarm events where applicable.  ..."

make "events" plural.  IMO.

Reply:
Updated.